stbt_auto_selftest.py

```diff
@@ -135,6 +139,9 @@ def generate_into_tmpdir(screenshots_root):
         for module_filename in _recursive_glob('*.py'):
             if module_filename.startswith('selftest'):
                 continue
+            if not is_valid_python_identifier(
+                    os.path.basename(module_filename)[:-3]):
+                continue
```

Comment:
This doesn't cause a problem for `inspect_module`.  The problem happens later when we try to fill in the doctest that we've created.  It would begin with `import bad-module-name` which is a `SyntaxError`.  Since I know it's going to fail I prefer to skip early to save importing it at all.